README.md

```diff
@@ -1,24 +1,31 @@
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/0a902c2fbf9e455f88c61bd0e75fed09)](https://app.codacy.com/gh/Seagate/cortx-fs-ganesha?utm_source=github.com&utm_medium=referral&utm_content=Seagate/cortx-fs-ganesha&utm_campaign=Badge_Grade_Settings)
 
-# CORTXFS-FSAL
+# CORTXFS-Ganesha
 
-NFS Ganesha backend for CORTXFS implementation.
+The CORTXFS-GANESHA repository provides the FSAL code for CORTXFS that is used by the CORTX-NFS-Ganesha repository. In other words, CORTXFS-Ganesha is the backend for CORTXFS implementation. The FSAL (Filesystem Abstraction Layer) implements the NFS-Ganesha FSAL-API based on CORTXFS as the underlying filesystem.
```

Comment:
Better wording would be:
The CORTXFS-GANESHA repository provides CORTXFS FSAL (Filesystem Abstraction Layer) code. This code implements NFS Ganesha FSAL-APIs, based on CORTXFS as the underlying filesystem. NFS Ganesha makes calls into CORTXFS FSAL code and CORTXFS FSAL code interacts with CORTXFS.

Reply:
Done. Codacy fails for all documents by default. you'll need to ignore the file for Codacy checks @sachinpunadikar